SpatialGDK/Documentation/toc.md

```diff
@@ -34,6 +34,8 @@
     - [Directory structure]({{urlRoot}}/content/directory-structure)
     - [Gameplay Ability System]({{urlRoot}}/content/ability-system)
     - [Non-Unreal server-worker types]({{urlRoot}}/content/non-unreal-server-worker-types)
+    - [SpatialOS Editor Settings]({{urlRoot}}/content/editor-settings)
```

Comment:
Suggest go with the Toolbar page under the title "Unreal Editor Interface" which matches the Unreal docs section on this: https://docs.unrealengine.com/en-US/Engine/UI